winui/Shimmer/Custom-Layout.md

```diff
@@ -0,0 +1,325 @@
+---
+layout: post
+title: Custom Layout in WinUI Shimmer control | Syncfusion
+description: Learn here all about the Custom Layout support in Syncfusion WinUI Shimmer (SfShimmer) control and more.
+platform: WinUI
```

Comment:
@GilbertmanojSF3971 

title: Custom layout in WinUI Shimmer control | Syncfusion
description: Learn here all about the custom layout support in Syncfusion WinUI Shimmer (SfShimmer) control to create your own shimmer type.

Reply:
@Manivannan-E Sir changed the title and description content.